game_cfg/colors.h

```diff
@@ -1,7 +1,5 @@
-#include <ncurses.h>
-
+// Text colors
```

Comment:
This is a pointless comment but won't be once I get more color constants in this section.